climada_petals/entity/exposures/test/test_openstreetmap.py

```diff
@@ -122,7 +42,7 @@ def test_osmqpiquery(self):
 
     def test_area_to_queryformat(self):
         """ test methods of OSMApiQuery"""
-
+        
```

Comment:
A lot of unnecessary whitespace added in this file. Please remove. Jenkins does not complain because our pylint does not parse the tests.
```suggestion

```